proxy/http/remap/PluginFactory.cc

```diff
@@ -0,0 +1,268 @@
+/** @file
+
+  A brief file description
```

Comment:
Sigh.

Reply:
Just copied the preamble somewhere from another `tscore` source file and did not notice that it had `@file` description on the top. Will merge with my comment bellow on line 24

We actually have more than 500 similar cases on master:

```
src/trafficserver $ grep -R 'A brief file description' *|wc -l
     522
```